gen/dcos-config.yaml

```diff
@@ -4,6 +4,7 @@ package:
       {
         "environment": {
           "MESOS_IP_DISCOVERY_COMMAND": "/opt/mesosphere/bin/detect_ip",
+          "PLATFORM": "{{ platform }}"
```

Comment:
Also, why does this need to be a global env var? Really trying to not make lots of global state (ends up turning into a microsoft registry type thing).